src/main/resources/jupyter/jupyter_notebook_config.py

```diff
@@ -28,7 +28,7 @@
 if os.environ['WELDER_ENABLED'] == 'true':
   # Only enable this along with Welder, as this change is backwards incompatible
   # for older localization users.
-  c.NotebookApp.notebook_dir = '/home/jupyter-user/notebooks'
+  c.NotebookApp.notebook_dir = $(notebooksDir)
```

Comment:
This could also be the EV: `${NOTEBOOKS_DIR}`. 

Doesn't really matter in this case but it might be useful anyway to set this EV inside the Jupyter container.